resources/user-guides/getting-started-with-adobe-experience-platform/sample-xdm-implementation.md

```diff
@@ -38,21 +38,23 @@ Set the `LAUNCH_ENVIRONMENT_FILE_ID` to the copied Environment File ID in the `M
    * Select **Create schema**, followed by **XDM ExperienceEvent**
    * Set the name for this schema as "Product Reviews".
    * Select the **Add** button (+) to add field groups
+     
      * Select **Environment Details** from the "Use existing field groups" section then select **Add field group**.
    * Select the **Add** button (+) to add field groups
+     
      * Select **Create new field group** and set the name as "Product review", then select **Add field group**.
-   * In the Schema structure, select the "Product review" field group and select the plus icon (+) next to the schema name "Product Reviews".
+* In the Schema structure, select the "Product review" field group and select the plus icon (+) next to the schema name "Product Reviews".
    * Start adding fields as follows. After each, select **Apply**, then plus icon (+) to add the next field.
-
+   
      | Field name | Display name | Type | Required |
      | :--- | :--- | :--- | :--- |
      | productSku | Product SKU | String | Yes |
      | reviewText | Review Text | String | No |
-     | rating | Rating | Integer | Yes |
+  | rating | Rating | Integer | Yes |
```

Comment:
Did you intend to remove this leading whitespace? At least in GH's rich text viewer, it causes this line to pop out of the table.